libraries/entities/src/DiffTraversal.cpp

```diff
@@ -153,7 +153,8 @@ void DiffTraversal::Waypoint::getNextVisibleElementDifferential(DiffTraversal::V
                                 distance2 = glm::distance2(lastView.viewFrustum.getPosition(), element->getAACube().calcCenter());
                                 if (distance2 >= visibleLimit * visibleLimit) {
                                     next.element = nextElement;
-                                    next.intersection = lastIntersection;
+                                    // element's intersection with lastView was effectively OUTSIDE
+                                    next.intersection = ViewFrustum::OUTSIDE;
                                     return;
                                 }
```

Comment:
This is the fix: when an element was culled for LOD in the lastView its "intersection" was effectively OUTSIDE.  There is special logic in subsequent "scan element for entities" code that will not bother scanning entities in an element whose "intersection" was INSIDE, but we want such a "previously-LOD-skipped" element to be scanned!